external/avro/src/main/scala/org/apache/spark/sql/avro/AvroFileFormat.scala

```diff
@@ -100,6 +77,50 @@ private[avro] class AvroFileFormat extends FileFormat
     }
   }
 
+  private def inferAvroSchemaFromFiles(
+      files: Seq[FileStatus],
+      conf: Configuration,
+      ignoreExtension: Boolean): Schema = {
+    val ignoreCorruptFiles = SQLConf.get.ignoreCorruptFiles
```

Comment:
Bout about matching it to `sparkSession.sessionState.conf.ignoreCorruptFiles` like other occurrences?